birdhouse/optional-components/scheduler-job-deploy_data/pre-docker-compose-up.include

```diff
@@ -0,0 +1,52 @@
+generate_scheduler_job_deploy_data_template() {
+  job_id="$1"
+  name="$(eval "echo \"\$SCHEDULER_JOB_${job_id}_DEPLOY_DATA_JOB_NAME\"")"
+  comment="$(eval "echo \"\$SCHEDULER_JOB_${job_id}_DEPLOY_DATA_JOB_COMMENT\"")"
+  schedule="$(eval "echo \"\$SCHEDULER_JOB_${job_id}_DEPLOY_DATA_JOB_SCHEDULE\"")"
+  config_file="$(eval "echo \"\$SCHEDULER_JOB_${job_id}_DEPLOY_DATA_JOB_CONFIG_FILE\"")"
+  checkout_cache="$(eval "echo \"\$SCHEDULER_JOB_${job_id}_DEPLOY_DATA_JOB_CHECKOUT_CACHE\"")"
+  log_file_name="$(eval "echo \"\$SCHEDULER_JOB_${job_id}_DEPLOY_DATA_JOB_LOG_FILENAME\"")"
+  image="$(eval "echo \"\$SCHEDULER_JOB_${job_id}_DEPLOY_DATA_JOB_DOCKER_IMAGE\"")"
+  git_ssh_id_file="$(eval "echo \"\$SCHEDULER_JOB_${job_id}_DEPLOY_DATA_JOB_GIT_SSH_ID_FILE\"")"
+  extra_options="$(eval "echo \"\$SCHEDULER_JOB_${job_id}_DEPLOY_DATA_JOB_EXTRA_OPTIONS\"")"
+
+  error_msg = "No XXX found for deploy data job with id '$job_id'"
+  [ -z "$name" ] && log ERROR "$(sed $error_msg | 's/XXX/name/')" && return 1
+  [ -z "$schedule" ] && log ERROR "$(sed $error_msg | 's/XXX/schedule/')" && return 1
+  [ -z "$config_file" ] && log ERROR "$(sed $error_msg | 's/XXX/config_file/')" && return 1
+
+  comment="${comment:-"${name}"}"
+  checkout_cache="${checkout_cache:-"${name}"}"
+  image="${image:-"${SCHEDULER_JOB_DEPLOY_DATA_JOB_IMAGE}"}"
+  git_ssh_id_file="${git_ssh_id_file:-"${DEPLOY_DATA_JOB_GIT_SSH_IDENTITY_FILE}"}"
+  log_file_name="${log_file_name:-"${name}.log"}"
```

Comment:
`log_file_name` default should be `${BIRDHOUSE_LOG_DIR}/${name}.log`, see https://github.com/bird-house/birdhouse-deploy/blob/b5cd7f6501793ea8e67f76d2208e02dc797ba031/birdhouse/components/scheduler/deploy_data_job.env#L45-L48